<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnTech::GuiQt::Entity::EntityRomStructListWidget</class>
 <widget class="QWidget" name="UnTech::GuiQt::Entity::EntityRomStructListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entity ROM Structs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="GuiQt::Accessor::NamedListView" name="listView"/>
   </item>
   <item>
    <widget class="QToolBar" name="buttons">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="floatable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuiQt::Accessor::NamedListView</class>
   <extends>QListView</extends>
   <header>gui-qt/accessor/namedlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
